src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,66 @@
+package mate.jdbc.model;
+
+import java.util.Objects;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
+    public Driver(){
+    }
+
+    public Driver(String name, String licenseNumber) {
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+    }
```

Comment:
If you use this constructor only once, it would be better to remove it, or use it not only once